packages/teleport/src/console/components/DocumentSsh/useSshSession.ts

```diff
@@ -91,10 +91,12 @@ function handleTtyConnect(
   session: Session,
   docId: number
 ) {
-  const { hostname, login, sid, clusterId } = session;
+  const { hostname, login, sid, serverId, clusterId } = session;
   const url = cfg.getSshSessionRoute({ sid, clusterId });
   ctx.updateSshDocument(docId, {
-    title: `${login}@${hostname}`,
+    // DELETE IN 5.0.0
+    // serverId is necessary for Teleport nodes <4.3
+    title: `${login}@${hostname || serverId}`,
```

Comment:
Same issue with missing `clusterName`, Teleport Nodes using < v4.3 has empty `hostname`.
We used `serverId` as a replacement for hostname in active sessions list, so I did the same for tabs name.
![server-hostname-issue](https://user-images.githubusercontent.com/43280172/87388913-8eb24e00-c55a-11ea-8432-4fe8008ce524.png)
![server-hostname-issue-2](https://user-images.githubusercontent.com/43280172/87388914-8f4ae480-c55a-11ea-8567-f007541f8ee9.png)

Reply:
revised to be handled in webapi server: https://github.com/gravitational/teleport/pull/4027